README.md

```diff
@@ -352,10 +352,32 @@ To support larger columns, you can use the `maxlength` column metadata field to
 
 Here is an example of updating a column's metadata field in Scala:
 
-```
+```scala
 import org.apache.spark.sql.types.MetadataBuilder
-val metadata = new MetadataBuilder().putLong("maxlength", 10).build()
-df.withColumn("colName", col("colName").as("colName", metadata)
+
+// Specify the custom width of each column
+val columnWidthMap = Map(
+  "column1" -> 10,
+  "column2" -> 11,
+  "column3" -> 12,
+  "column4" -> 13,
+  "column5" -> 14,
+)
+
+var df // the dataframe you'll want to write to redshift
+
+// Apply each column metadata customization
+columnWidthMap.foreach(m => {
```

Comment:
Style nit: you could write `columnWidthMap.foreach { m =>` in order to omit an extra layer of braces.